turn_alerts/views.py

```diff
@@ -0,0 +1,123 @@
+import re
+import time
+
+from prometheus_client import Counter, Histogram
+from rest_framework import generics, permissions, status
+from rest_framework.response import Response
+
+from turn_alerts.serializers import (
+    ContactsPayloadSerializer,
+    Status_ErrorPayloadSerializer,
+    StatusPayloadSerializer,
+    VendorPayloadSerializer,
+)
+
+from .tasks import start_turn_journey
+
+# Prometheus counters for events and messages
+message_requests_total = Counter(
+    "turn_alerts_message_requests_total",
+    "Total number of message requests",
+    [
+        "fallback_channel",
+        "direction",
+        "message_type",
+        "message_body",
+        "state_reason",
+        "state",
+        "chat_id",
+        "labels",
+        "assigned_to",
+        "permalink",
+    ],
+)
+
+
+event_count = Counter(
+    "turn_alerts_event_count",
+    "Number of events",
+    ["message_status", "error_code", "error_data"],
+)
+
+successful_event_count = Counter(
+    "turn_alerts_successful_event_count",
+    "Number of events",
+    ["message_type", "message_status", "conversation_id"],
+)
+
+
+processing_time = Histogram(
```

Comment:
Don't think we need this

Reply:
Funtionally no but I added this just to see how the buckets thing works to split the proessing times. I can remove.